homeassistant/loader.py

```diff
@@ -900,6 +900,14 @@ def get_component(self) -> ComponentProtocol:
             )
             raise ImportError(f"Exception importing {self.pkg_path}") from err
 
+        if self.platform_exists("config") is not False:
```

Comment:
```suggestion
        if self.platform_exists("config"):
```

Can't return `None` here since we know the comp is loaded